core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -390,4 +394,70 @@ fun LPGenericElevatedCardImage(
             }
         }
     }
-}
\ No newline at end of file
+}
+/**
+ * [LpPaymentsRentCard] is a button for show in the UI and this button can have icon depending of the use
+ *
+ * @param modifier Modifier is for specify style and params of the card, like for example the width of the component.
+ * @param month is the month what this component shown at the user for indicate what month is must to pay
+ * @param quantity is the quantity what the user have to pay
+ * @param date is the limit date where it have to pay their rent
+ *
+ * @author Usiel Filiberto Garcia Jimenez
+ */
+@Composable
+fun LpPaymentsRentCard(modifier: Modifier, month: String, quantity: Double, date: Date) {
+    OutlinedCard(
+        modifier = modifier
+            .fillMaxWidth(),
+        colors = CardDefaults
+            .cardColors(
+                containerColor = Color.Transparent
+            ),
+        border = BorderStroke(
+            width = LeasePertTheme.sizes.stroke,
+            color = MaterialTheme.colorScheme.onPrimary
+        )
+    ) {
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .padding(start = 15.dp, top = 15.dp, bottom = 5.dp)
+        ) {
+            Text(
+                text = month,
+                style = MaterialTheme.typography.bodyLarge)
+        }
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .padding(start = 15.dp, bottom = 15.dp, end = 15.dp),
+
+            ) {
+            Text(
+                text = stringResource(R.string.rent),
+                style = MaterialTheme.typography.bodyLarge,
+                color = MaterialTheme.colorScheme.tertiary
+            )
+            Column(modifier = Modifier.padding(start = 45.dp)) {
+                Text(
```

Comment:
You don't need to add the modifer if you aren't going to use at least one property.

Reply:
Fixed